tests/cloud_playbooks/create-gce.yml

```diff
@@ -35,7 +35,7 @@
       register: gce
 
     - name: Add instances to host group
-      add_host: hostname={{item.name}} ansible_host={{item.public_ip}} groupname="waitfor_hosts"
+      add_host: hostname={{item.public_ip}} groupname="waitfor_hosts"
```

Comment:
isn't this also:  https://github.com/kubernetes-incubator/kubespray/pull/1646 ?

Reply:
yeah I'll get rid of it